packages/storage-plus/src/keys.rs

```diff
@@ -79,7 +79,9 @@ impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a>> PrimaryKey<'a> for
 impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a> + Prefixer<'a>, V: PrimaryKey<'a>>
     PrimaryKey<'a> for (T, U, V)
 {
-    type Prefix = T;
+    // FIXME: Generalize
+    // type Prefix = (T, U);
+    type Prefix = Pk2<'a>;
```

Comment:
This is wrong - we change types magically.

You could do `Pk2<'a, T, U>`, but we need to make sure the T + U are preserved.
This likely means updating `impl Prefixer for Pk2` definition

Reply:
Yes, I know. I couldn't come up with a proper impl. Will revisit.